test/conftest.py

```diff
@@ -26,3 +27,35 @@ def mock_pdf(mock_pdf_stream, tmpdir_factory):
     with open(fn, 'wb') as f:
         f.write(file.getbuffer())
     return str(fn)
+
+
+@pytest.fixture(scope="session")
+def mock_image_stream():
+    url = "https://miro.medium.com/max/3349/1*mk1-6aYaf_Bes1E3Imhc0A.jpeg"
+    return requests.get(url).content
+
+
+@pytest.fixture(scope="session")
+def mock_image_folder(mock_image_stream, tmpdir_factory):
+    file = BytesIO(mock_image_stream)
+    folder = tmpdir_factory.mktemp("images")
+    for i in range(5):
+        fn = folder.join("mock_image_file_" + str(i) + ".jpeg")
+        with open(fn, 'wb') as f:
+            f.write(file.getbuffer())
+    return str(folder)
+
+
+@pytest.fixture(scope="session")
+def mock_detection_label(tmpdir_factory):
+    folder = tmpdir_factory.mktemp("labels")
+    label = {
+        "boxes_1": [[[1, 2], [1, 3], [2, 1], [2, 3]], [[10, 20], [10, 30], [20, 10], [20, 30]]],
+        "boxes_2": [[[3, 2], [3, 3], [4, 1], [4, 3]], [[30, 20], [30, 30], [40, 10], [40, 30]]],
+        "boxes_3": [[[1, 5], [1, 8], [2, 5], [2, 8]]],
+    }
+    for i in range(5):
+        fn = folder.join("mock_image_file_" + str(i) + ".jpeg.json")
+        with open(fn, 'w') as f:
+            json.dump(label, f)
+    return str(folder)
```

Comment:
This part should be moved in `test_datasets_loaders.py` for clarity (it won't be used by others, while the image folder could be useful elsewhere)